src/index.html

```diff
@@ -263,9 +297,59 @@ <h2>QUICK ID</h2>
           </table>
         </section>
 
-        <section>
+        <section style="background-color:lightgray ">
           <h2>CONTACT ME</h2>
           <!-- Form Related Content for Lesson 2 -->
+          <from>
+            <fieldset>
```

Comment:
Please add all required styles in `src/css/index.scss`
Lesson 2 assignment require you to style the quick id and contact me section.